doc/0000-ccl_note/main.tex

```diff
@@ -1186,9 +1232,11 @@ \section{Future functionality to be included}
 
 In the future, we hope that \ccl will include other functionalities. Functionalities which are currently under development:
 \begin{itemize}
-        \item a link to {\tt angpow} \citep{2017arXiv170103592C} for going beyond the Limber approximation,
-	\item a link to {\tt FAST-PT} \citep{FASTPT} for efficient implementation of nonlinear perturbation theory,
-	\item and more power spectrum methods (see \ref{Pk_wishlist}).
+\item further CMB observable predictions (beyond CMB lensing)\footnote{\url{https://github.com/LSSTDESC/CCL/issues/428}},
+\item a link to {\tt FAST-PT} \citep{FASTPT} for efficient implementation of nonlinear perturbation theory,
+\item halo density profiles\footnote{\url{https://github.com/LSSTDESC/CCL/issues/407}}
+\item linear-theory only predictions\footnote{\url{https://github.com/LSSTDESC/CCL/issues/438}}.
+\item and more power spectrum methods (see \ref{Pk_wishlist}).
 \end{itemize}
```

Comment:
Other possible things to add:
- Velocity correlation function: https://github.com/LSSTDESC/CCL/issues/429
- Mass function emulators: https://github.com/LSSTDESC/CCL/issues/235

Reply:
Added